client-ts/Microsoft.AspNetCore.SignalR.Client.TS/HubConnection.ts

```diff
@@ -224,9 +224,18 @@ class Base64EncodedHubProtocol implements IHubProtocol {
     readonly type: ProtocolType;
 
     parseMessages(input: any): HubMessage[] {
+
+        // The format of the message is `size:message;`
+        let pos = input.indexOf(":");
+        if (pos == -1)
+        {
+            throw new Error("Invalid payload.");
+        }
+        let encodedMessage = input.substring(pos + 1, input.length - 1);
```

Comment:
We don't batch messages for SSE and at the moment we are guaranteed to receive the entire message. If the message is incomplete base64 decoding will fail. Anyways, I added the checks for completeness for now. The length prefix will be used for real when we move to pipelines - then it will be possible to receive incomplete messages.